bytes/bytes.ts

```diff
@@ -0,0 +1,54 @@
+export function bytesFindIndex(a: Uint8Array, pat: Uint8Array): number {
+  const s = pat[0];
+  for (let i = 0; i < a.length; i++) {
+    if (a[i] !== s) continue;
+    let pin = i;
```

Comment:
`pin` is not reassigned, should be `const`

Reply:
We need tslint in deno_std...